vespalib/src/tests/stllike/hashtable_test.cpp

```diff
@@ -69,9 +67,24 @@ TEST("require that hashtable<int> can be copied") {
     EXPECT_EQUAL(42, *table2.find(42));
 }
 
+TEST("require that getModuloStl always return a larger number in 32 bit integer range") {
+    for (size_t i=0; i < 32; i++) {
+        size_t num = 1ul << i;
+        size_t prime = hashtable_base::getModuloStl(num);
+        EXPECT_GREATER_EQUAL(prime, num);
+        printf("%lu <= %lu\n", num, prime);
+    }
+    for (size_t i=0; i < 32; i++) {
+        size_t num = (1ul << i) - 1;
+        size_t prime = hashtable_base::getModuloStl(num);
+        EXPECT_GREATER_EQUAL(prime, num);
+        printf("%lu <= %lu\n", num, prime);
+    }
+}
```

Comment:
Perhaps also test all the primes, and their ++1?

Reply:
Done